src/commands/bundle.ts

```diff
@@ -0,0 +1,81 @@
+import { Flags } from '@oclif/core';
+import { Example } from '@oclif/core/lib/interfaces';
+import Command from '../base';
+import bundle from '@asyncapi/bundler';
+import fs from 'fs';
+import path from 'path';
+import { ErrorLoadingSpec } from '../errors/specification-file';
+
+export default class Bundle extends Command {
+  static description = 'Bundle multiple asyncapi files together.';
+  static strict = false;
+
+  static examples: Example[] = [
+    'asyncapi bundle ./asyncapi.yaml -o final.yaml',
+    'asyncapi bundle ./asyncapi.yaml ./spec.yaml --reference-into-components'
+  ];
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+    output: Flags.string({ char: 'o', description: 'The output file name. Omitting this flag will create a main.yaml.' }),
+    'reference-into-components': Flags.boolean({ char: 'r', description: 'Bundle the message $refs into components.' }),
+    base: Flags.string({ char: 'b' , description: 'Path to the file which will act as a base. This is required when some properties are to needed to be overwritten.'}),
```

Comment:
```suggestion
    base: Flags.string({ char: 'b' , description: 'Path to the file which will act as a base. This is required when some properties are to needed to be overwritten.'}),
```

what do you mean by this description? `This is required when some properties are to needed to be overwritten.`

Reply:
good to have example for it in help

Reply:
I also haven't seen a test for `-b` or `--base` flag

Reply:
> what do you mean by this description? This is required when some properties are to needed to be overwritten.

For example when you have two separate files and after combining you want to have a separate new info object or add a new license, you can pass a object that would be used to change those things as per required. 

I am currently testing it, I am facing some issues as it is not working as intended with the new `reference-into-components` feature.

Reply:
so maybe for now remove it from PR so it doesn't block you?

Reply:
Already fixed, it in the bundler